nmtrust

```diff
@@ -70,26 +81,31 @@ while getopts ":f:qh" opt; do
     esac
 done
 
-# Check if the trusted network file exists.
+# Check if the excluded and trusted networks files exist.
 file_check
 
 # Get all active connections.
-connections=($(nmcli --terse -f uuid conn show --active))
+connections=($(nmcli --terse -f name,uuid conn show --active))
+
+# Get number of active connections.
+num_connections=$(comm -13 <(sort "$EXCLUDEFILE") <(nmcli --terse -f name conn show --active | sort) | wc -l)
 
 # Get number of trusted connections.
 num_trusted=$(comm -12 <(nmcli --terse -f uuid conn show --active | sort) <(sort "$TRUSTFILE") | wc -l)
 
 # Determine if there are active connections.
-if [ ${#connections[@]} -eq 0 ]; then
+if [ $num_connections -eq 0 ]; then
     no_network
 # Check if any of the active connections are untrusted.
 elif [[ $num_trusted -eq 0 ]]; then
     all_untrusted
 else
-    for uuid in "${connections[@]}"; do
-        if ! grep -q ^"$uuid"$ "$TRUSTFILE"; then
-            num_untrusted=$((${#connections[@]} - num_trusted))
-            untrusted "$num_untrusted of ${#connections[@]}"
+    for connection in "${connections[@]}"; do
+        name=$(echo "$connection" | awk -F ":" '{print $1}')
+        uuid=$(echo "$connection" | awk -F ":" '{print $2}')
+        if ! grep -q ^"$name"$ "$EXCLUDEFILE" && ! grep -q ^"$uuid"$ "$TRUSTFILE"; then
```

Comment:
It looks like this expects the exclusion files to list networks by name, which requires the network names always to be unique. Does that make sense? It would probably be more intuitive for the exclusion and trusted network file to have the same format.

Reply:
I want to specify the network by name because the UUID will most probably change after re-installation (of Docker or Arch) and I would need to update my automation every time. 

Using the the name makes sense in case of local bridge networks like Docker bridges `docker0`, `docker1`, etc. pp. However, it doesn't make sense to specify external networks (by UUID) in this file since this would defeat the purpose of nmtrust.

Reply:
Ok, let's keep it as name.